server/App.js

```diff
@@ -1,11 +1,13 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import morgan from 'morgan';
+import router from './routes/IndexRoute';
```

Comment:
The default export should match the base file name. check out Airbnb style guide.